deployment/copilot/README.md

```diff
@@ -186,5 +190,16 @@ Official documentation on Addons can be found [here](https://aws.github.io/copil
 
 ### Useful Commands
 
-`copilot app show`: Provides details on the current app \
-`copilot svc show`: Provides details on a particular service
+`copilot app show` - Provides details on the current app \
+`copilot svc show` - Provides details on a particular service
+
+### Removing deloyed resources from AWS
+
+To remove the resources installed from the steps above, follow these instructions:
+1. Access the AWS Console and delete all S3 objects and content within the "StagingBucket".
+2. Execute the following commands:
+    ```
+    ./devDeploy.sh --destroy-env
+    ./devDeploy.sh --destroy-all-copilot`
+    ```
+**Warning**: If Copilot resources were deployed across multiple regions, some resources may need to be removed manually. This can be achieved by deleting the CloudFormation stacks in each respective region. Ensure to look for resources tagged with the key migration_deployment and a value corresponding to the migration version.
```

Comment:
This didn't make sense to me, what stacks would be left after running the above commands?

Reply:
I removed warning for the time being.